MaterialDesignThemes.Wpf.Tests/RatingBarTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Xunit;
+
+namespace MaterialDesignThemes.Wpf.Tests;
+
+public class RatingBarTests
+{
+    [StaTheory]
+    [InlineData(-5, 0.0)]
```

Comment:
Should the expected value here be 1.0 to match the Min?

Reply:
Well hidden in my commit comments, I left this little comment:

    "It is worth noting that valid values are actually values between Min-1 and Max (in multiples of 0.5)"

I agree this is somewhat strange, and I will work on changing the behavior.

It should however be noted that the implementation before this PR **did not** have any coercion (which is strange for a "range bound" control IMO), and thus allowed you to set **any** value.

In any case, the first button represents the `Min` value today, and that is a little strange because that actually means you should not be able to set a rating less than 1 star when `Min`=1; why should it not be allowed to set 0 stars (although difficult to do with the mouse!)? Anyway, I will see if I can rework this into something a little more intuitive while preserving back-compat.